components/Main.js

```diff
@@ -32,6 +44,26 @@ const UI_STRINGS = {
 }
 
 export default function Main() {
+  const [hasPoppinsLoaded] = usePoppins({
+    Poppins_700Bold,
+  })
+
+  const [hasDidactLoaded] = useDidactGothic({
+    DidactGothic_400Regular,
+  })
```

Comment:
we're not supposed to be loading fonts twice. this is already happening in App.js and should not happen again. App initialization should live in a single place at the root of the component tree

Reply:
Good catch! The font loading on App.js was a left over. I had to move everything out to Main.js to be able to couple them with the secret storage initialization. I'm cleaning up that up now.